### DISCUSSION TITLE
v3.11.0

### DISCUSSION THREAD

**Discussion by Ethosa** [HapticX/happyx#268] [Announcements]
# Functional components 🔥
## What's Changed
* v3.11.0 - functional components by @Ethosa in https://github.com/HapticX/happyx/pull/267
* little fix for components by @Ethosa in https://github.com/HapticX/happyx/pull/264

**Full Changelog**: https://github.com/HapticX/happyx/compare/v3.10.2...v3.11.0

<hr /><em>This discussion was created from the release <a href='https://github.com/HapticX/happyx/releases/tag/v3.11.0'>v3.11.0</a>.</em>

**Comment by lightcax** [comment]
Will the database orm be added in the future?
